src/compression.cs

```diff
@@ -0,0 +1,6 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Compression {
+
+}
```

Comment:
Is that needed ?

Reply:
We get compilation errors because we do not have the file. I could remove the namespace.

Reply:
Yes, the file is required, but it can be empty.